backend-server/src/main/java/org/dice/ida/controller/MessageController.java

```diff
@@ -89,7 +88,8 @@ public String greeting() {
 				response.setMessage(IDAConst.BOT_SOMETHING_WRONG);
 				response.setUiAction(IDAConst.UAC_NRMLMSG);
 			}
-			chatLog.info("session id:\t" + idaChatBot.fetchDfSessionId() + "\t Response:\t" + response.getMessage());
+			chatLog.info("Context list: " + dialogFlowUtil.getActiveContextList());
```

Comment:
We do not just want to see the list of active contexts, but also which intent was this particular message assigned to by Dialogflow.